lib/zendesk_apps_support/validations/manifest.rb

```diff
@@ -66,6 +55,37 @@ def collate_manifest_errors(package)
           errors.flatten.compact
         end
 
+        def validate_location(package)
+          manifest = package.manifest
+          errors = []
+          errors << missing_location_error(package)
+          errors << invalid_location_error(package)
+          errors << invalid_v1_location(package)
+          errors << location_framework_mismatch(manifest)
```

Comment:
I would like to see `errors` explicitly returned at the end of this method. It might be returned by `<<` but if someone modified this method and added some additional logic they might be surprised when their code stops working. Having `return errors` (or rubocop will probably make you write it as just `errors`) will make the method a little more resilient to change.

Reply:
done!